app/components/c-product-detail.vue

```diff
@@ -61,13 +62,22 @@
          *
          * @returns  {Object}  product - Single product from the store
          */
-        product: 'product/getProduct'
+        product: 'product/getProduct',
+
+        /**
+         * Gets a product
+         *
+         * @returns  {Object}  product - Single product from the store
+         */
+        productInformation: 'product/getProductInformation',
```

Comment:
- [ ] I think we should not use `get...` for the getters, because they will all start with this. Better just use the name of the data they return. `productInformation` in this case. I don't think they will be confused with the state, since you would have to call `this.$store.state....`